docs/export/README.md

```diff
@@ -0,0 +1,100 @@
+Netdata metrics to more than 20 different endpoints, including every [service that
+supports](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage) Prometheus remote write. See
+the [exporting reference guide](/exporting/README.md) for the full list.
+
+## Exporting quickstart
+
+Let's cover the process of enabling an exporting connector, using the Graphite connector as an example. These steps can
+be applied to other connectors as well.
+
+> If you are migrating from the deprecated backends system, this quickstart will also help you update your configuration
+> to the new format.
+
+Open the `exporting.conf` file with `edit-config`.
+
+```bash
+cd /etc/netdata   # Replace this path with your Netdata config directory, if different as found in the steps above
+sudo ./edit-config exporting.conf
+```
+
+### Enable the exporting engine
+
+Enable the exporting engine by setting `enabled` to `yes`:
+
+```conf
+[exporting:global]
+    enabled = yes
+```
+
+### Change how often the exporting engine sends metrics
+
+By default, the exporting engine only sends metrics to external databases every 10 seconds to avoid congesting the
+destination with thousands of per-second metrics.
+
+You can change this frequency for all connectors based on how you use exported metrics or the resources you can allocate
+to long-term storage. Use the `update every` setting to change the frequency in seconds.
+
+```conf
+[exporting:global]
+    update every = 10
+```
+
+### Enable a connector (Graphite)
+
+To enable the Graphite connector, find the `[graphite:my_graphite_instance]` example section in `exporting.conf`. You
+can use this (or the respective example for the connector you want to use) as a framework for your configration.
+Uncomment the section itself and replace `my_graphite_instance` with a name of your choice. Then set `enabled` to `yes`
+and uncomment the line.
+
+```conf
+[graphite:my_graphite_instance]
+    enabled = yes
+    # destination = localhost:2003
+    # data source = average
+    # prefix = netdata
+    # hostname = my_hostname
+    # update every = 10
+    # buffer on failures = 10
+    # timeout ms = 20000
+    # send names instead of ids = yes
+    # send charts matching = *
+    # send hosts matching = localhost *
+```
+
+Next, edit and uncomment any other lines necessary to connect the exporting engine to your endpoint. You must change the
+`destination` setting in most situations.
+
+For details on all the configuration options, see the [exporting reference](/exporting/README.md#configuration).
+
+> When migrating from backends, keep in kind that most settings for the exporting engine are identical to their backend
+> counterpart. You should be able to copy your backends settings into `exporting.conf` to complete your migration. The
+> only setting that won't work is `host tags`. Instead, use [host labels](/docs/tutorials/using-host-labels.md) to tag
```

Comment:
Should we mention here anything about the `type` option and section names, like in https://github.com/joelhans/netdata/blob/exporting/exporting/README.md#L24?

Reply:
Good call. I added a mention earlier in the document.